### PR TITLE
COG-660 Switch between success and error alerts

### PR DESCRIPTION
**Pull request by Nasir2712**
#### Which User story does this relate to (link)?
-https://osi-cwds.atlassian.net/browse/COG-660

#### Brief feature description
- Switch between success alerts and error alerts in user detail page

### CODE REVIEW

File: app/javascript/views/userDetail/UserDetail.jsx [hunk 1]
```diff
@@ -182,6 +182,7 @@ export default class UserDetail extends Component {
       permissionsList,
       userDetailError,
     } = this.props;
+    const { alert } = this.state;
```
[the remark this answers is not in the document]
Comment (Nasir2712) [reply app/javascript/views/userDetail/UserDetail.jsx]: nothing. I just wanted to follow the same destructuring pattern as props for the state as well.

[the remark this answers is not in the document]
Comment (kelfink) [reply app/javascript/views/userDetail/UserDetail.jsx]: ?? I'm more confused now.  I assumed it was a boolean.  Nothing ?

[the remark this answers is not in the document]
Comment (Nasir2712) [reply app/javascript/views/userDetail/UserDetail.jsx]: i thought you were asking thinking it was an object. my bad. Yes, it is boolean.

File: app/javascript/views/userDetail/UserDetail.test.js [hunk 2]
```diff
@@ -226,19 +216,30 @@ describe('UserDetail', () => {
...
+      wrapper.setProps({ userDetailError: null });
+      const alertBox = wrapper.find('Alert');
+      expect(alertBox.length).toBe(1);
+      expect(alertBox.props().children).toBe(
+        'Your changes have been made successfully'
+      );
+      expect(alertBox.props().alertCross).toEqual(false);
+    });
+
     it('does not display error <Alert/>', () => {
```
Comment (RamuKammagani-osi) [inline app/javascript/views/userDetail/UserDetail.test.js]: i think you need to rephrase this description, "does not display error Alert" isn't it doesn't display success Alert ??

Comment (Nasir2712) [reply app/javascript/views/userDetail/UserDetail.test.js]: ya, I already have another description with success alert, I'll rename it to just alert.

Comment (RamuKammagani-osi) [reply app/javascript/views/userDetail/UserDetail.test.js]: 👍

**Pull request merged**